py/kubeflow/testing/create_kf_instance.py

```diff
@@ -94,11 +94,16 @@ def deploy_with_kfctl_go(kfctl_path, args, app_dir, env):
   # loading the config in the repo we have checked out kfctl doesn't support
   # specifying a file URI. Once it does we should change --version to
   # use it.
+  #
+  # TODO(zhenghuiwang): use the master of kubeflow/manifests once
+  # https://github.com/kubeflow/kubeflow/issues/3475 is fixed.
```

Comment:
@richardsliu is fixing this right? Should we just wait for it?

Reply:
I didn't close the the issue. I will make a PR using master once it is fixed.

This may or may not fix the cron job. The easy way to test it is to merge this PR..